addons/keda/locals.tf

```diff
@@ -0,0 +1,43 @@
+locals {
+  name = "keda"
+
+  default_helm_config = {
+    name                       = try(var.keda_extra_configs.name, local.name)
+    chart                      = try(var.keda_extra_configs.chart, local.name)
+    repository                 = try(var.keda_extra_configs.repository, "https://kedacore.github.io/charts")
+    version                    = try(var.keda_extra_configs.version, "2.11.2")
+    namespace                  = try(var.keda_extra_configs.namespace, "keda")
+    create_namespace           = try(var.keda_extra_configs.create_namespace, true)
+    description                = "Keda helm Chart deployment configuration"
+    timeout                    = try(var.keda_extra_configs.timeout, "600")
+    lint                       = try(var.keda_extra_configs.lint, "false")
+    repository_key_file        = try(var.keda_extra_configs.repository_key_file, "")
+    repository_cert_file       = try(var.keda_extra_configs.repository_cert_file, "")
+    repository_username        = try(var.keda_extra_configs.repository_password, "")
+    repository_password        = try(var.keda_extra_configs.repository_password, "")
+    verify                     = try(var.keda_extra_configs.verify, "false")
+    keyring                    = try(var.keda_extra_configs.keyring, "")
+    disable_webhooks           = try(var.keda_extra_configs.disable_webhooks, "false")
+    reuse_values               = try(var.keda_extra_configs.reuse_values, "false")
+    reset_values               = try(var.keda_extra_configs.reset_values, "false")
+    force_update               = try(var.keda_extra_configs.force_update, "false")
+    recreate_pods              = try(var.keda_extra_configs.recreate_pods, "false")
+    cleanup_on_fail            = try(var.keda_extra_configs.cleanup_on_fail, "false")
+    max_history                = try(var.keda_extra_configs.max_history, "0")
+    atomic                     = try(var.keda_extra_configs.atomic, "false")
+    skip_crds                  = try(var.keda_extra_configs.skip_crds, "false")
+    render_subchart_notes      = try(var.keda_extra_configs.render_subchart_notes, "true")
+    disable_openapi_validation = try(var.keda_extra_configs.disable_openapi_validation, "false")
+    wait                       = try(var.keda_extra_configs.wait, "true")
+    wait_for_jobs              = try(var.keda_extra_configs.wait_for_jobs, "false")
+    dependency_update          = try(var.keda_extra_configs.dependency_update, "false")
+    replace                    = try(var.keda_extra_configs.replace, "false")
+  }
+
+  keda_extra_configs = var.keda_extra_configs
```

Comment:
This can be removed, as we are not using `local.keda_extra_configs` anywhere

Reply:
done